compiler/luci-interpreter/src/kernels/CMakeLists.txt

```diff
@@ -115,6 +115,8 @@ set(SOURCES
     StridedSlice.cpp
     Sqrt.h
     Sqrt.cpp
+    Square.h
+    Square.cpp
```

Comment:
Please add `Sqaure.test.cpp` in the same file.

Reply:
Thanks! forgot to copy-paste -_-;